### PR TITLE
Reconcile with new HTTPS page and clarify

### PR DESCRIPTION
**Pull request by Ordinant**

### CODE REVIEW

File: docs/src/main/sphinx/admin/web-interface.rst [hunk 1]
```diff
@@ -16,27 +16,28 @@ Authentication
...
+allowed. Typically, users login with the same username that they use for
 running queries.
 
-Accessing the Web UI over HTTPS requires configuring an authentication type for
-the Web UI or the Trino server. If no authentication type is configured for the
-Web UI, then it will chosen based on the Trino server authentication type.
+Accessing the Web UI over HTTPS requires configuring :doc:`TLS support
+</security/https>` for the Trino server, or independently configuring an
+authentication type for the Web UI. If no authentication type is configured for
+the Web UI, then it is chosen based on the Trino server authentication type.
```
Comment (dain) [inline docs/src/main/sphinx/admin/web-interface.rst:25]: I'd drop this paragraph entirely.  When these docs were written the WebUI on HTTPS required authentication and that is no longer true, and generally just works, so I would not call out anything special here as it is now more confusing than helpful

**Review by dain — COMMENTED** [review]
one comment but otherwise looks good to me

File: docs/src/main/sphinx/admin/web-interface.rst (hunk 1 above)
Comment (Ordinant) [reply docs/src/main/sphinx/admin/web-interface.rst:25]: done

**Review by mosabua — APPROVED** [review]
Looks good.

File: docs/src/main/sphinx/admin/web-interface.rst [hunk 2]
```diff
@@ -16,27 +16,28 @@ Authentication
...
 authentication, then any username can be used, and no password is required or
-allowed. Typically, users should login with the same username that they use for
+allowed. Typically, users login with the same username that they use for
 running queries.
 
-Accessing the Web UI over HTTPS requires configuring an authentication type for
-the Web UI or the Trino server. If no authentication type is configured for the
-Web UI, then it will chosen based on the Trino server authentication type.
+Accessing the Web UI over HTTPS requires configuring :doc:`TLS support
+</security/https>` for the Trino server, or independently configuring an
```
[the remark this answers is not in the document]
Comment (Ordinant) [reply docs/src/main/sphinx/admin/web-interface.rst]: That whole paragraph was dropped per resolved comment from Dain, above. It referred to the use of the old ``web-ui.authentication.type`` property, where you could specify an auth type for the Web UI that was independent of the auth type for your Trino server. But nowadays, you just get TLS working, then auth works.

**Review by electrum — APPROVED** [review]
Please remove the commit description (not needed) and update the title, since this doesn't link to the new page.

**Comment by electrum** [comment]
@Ordinant Please update the commit message using `git commit --amend` and then do a force push

**Comment by Ordinant** [comment]
Commit title changed per request

**Comment by electrum** [comment]
Thanks!

**Pull request merged**